crates/storage/src/column.rs

```diff
@@ -71,6 +71,12 @@ pub enum Column {
     // TODO: Remove this column and use `Metadata` column instead.
     /// Table for genesis state import progress tracking.
     GenesisMetadata = 21,
+    /// The Merkle Nodes for the Coins Table
+    #[cfg(feature = "global-state-root")]
+    CoinsMerkleData = 22,
+    /// The Merkle Root for the Coins Table
+    #[cfg(feature = "global-state-root")]
+    CoinsMerkleMetadata = 23,
```

Comment:
For Global Merkle root we can create a new `Column` type. We don't need to re-use onchain column type

Reply:
Done in https://github.com/FuelLabs/fuel-core/pull/2553/commits/13521ac08c857a093836a9bbf02f43986099a89c (by you ;) )